tezos_interop/tezos_interop.re

```diff
@@ -11,6 +11,16 @@ let rec try_decode_list = (l, string) =>
   | [] => None
   };
 
+let blake2b_20_encoding =
```

Comment:
I was a little confused about this at first, is the size 20 bytes just because that's what's used by tezos here https://gitlab.com/tezos/tezos/-/blob/master/src/lib_crypto/ed25519.ml#L40 ?

Reply:
Yup, key_hash should always be 160bits, that's a tradition since bitcoin. Our default blake2b is the 256bits one

Reply:
Perhaps document this in the rei file?
https://github.com/marigold-dev/sidechain/blob/1d9c49c856bb7b7eac5dfa773b23d80fd3e836fb/tezos_interop/tezos_interop.rei#L14

Reply:
This module models tezos data types, so any decision here should be "why is 20 bytes in Tezos?".